doc/kyua.1.in

```diff
@@ -300,7 +307,7 @@ Also, please include a copy of the log file corresponding to the problem
 you are experiencing.
 Unless you have changed the location of the log files, you can most likely
```

Comment:
This is potentially misleading after this change, as the location isn't clearcut if `$XDG_CACHE_HOME` is not set.